index.html

```diff
@@ -31,6 +31,138 @@ <h1 id="main-heading">Hey there. I'm Ulises <br><strong>I'm a Software Developer
+      <h2 class="headings-2">My Recent Works</h2>
+      <hr id="border-line">
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
   </section>
+
+  <article id="about-me-article">
+    <h2 class="headings-2">About Me</h2>
+    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
```

Comment:
- [ ] [REQUIERD] Please, add background image for the about me section as mentioned in the project requirements.

Figma Design | Your Design
---------|------------
![image](https://user-images.githubusercontent.com/60979926/164780117-d00d6c53-2a33-4985-8a58-becbc2f110e7.png) | ![image](https://user-images.githubusercontent.com/60979926/164780179-a5b9bb96-e95e-4c74-b5e9-6f0c11506c9e.png)